vms/platformvm/state/state.go

```diff
@@ -244,63 +238,54 @@ type state struct {
 	currentStakers *baseStakers
 	pendingStakers *baseStakers
 
-	delegateeRewardCache    map[ids.NodeID]map[ids.ID]uint64 // (nodeID, subnetID) --> delegatee amount
-	modifiedDelegateeReward map[ids.NodeID]set.Set[ids.ID]   // tracks (nodeID, subnetID) pairs updated after last commit
+	// Node ID --> Subnet ID --> Delegatee Reward on that subnet for that node ID.
+	modifiedDelegateeReward map[ids.NodeID]map[ids.ID]uint64
 
 	// UTXOs section
-	modifiedUTXOs map[ids.ID]*avax.UTXO            // map of UTXO ID -> *UTXO
-	utxoCache     cache.Cacher[ids.ID, *avax.UTXO] // UTXO ID -> *UTXO. If the *UTXO is nil the UTXO doesn't exist
+	modifiedUTXOs map[ids.ID]*avax.UTXO // map of UTXO ID -> *UTXO
 
 	// Metadata section
 	chainTime, latestComittedChainTime                  time.Time
```

Comment:
@danlaine do we really need latestComittedChainTime and latestCommittedLastAcceptedBlkID? I can't recall if we do

Reply:
They can save us from doing unnecessary writes but they'll probably be removed in a subsequent PR as we change how we commit this data to merkledb (i.e. with views)